runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/streaming/ActiveWorkState.java

```diff
@@ -83,6 +85,33 @@ static ActiveWorkState forTesting(
     return new ActiveWorkState(activeWork, computationStateCache);
   }
 
+  private static Stream<KeyedGetDataRequest> toKeyedGetDataRequests(
+      ShardedKey shardedKey, Collection<Work> workQueue, Instant refreshDeadline) {
+    return workQueue.stream()
+        .filter(work -> work.getStartTime().isBefore(refreshDeadline))
+        .map(
+            work ->
+                Windmill.KeyedGetDataRequest.newBuilder()
+                    .setKey(shardedKey.key())
```

Comment:
can rm the key, sharding key is sufficient and cheaper

Reply:
could do separately (and possibly in other places where we make reads or commits) since this is just moved